src/pages/Environments.js

```diff
@@ -202,23 +202,26 @@ const Environments = () => {
   const disksByProject = _.groupBy('googleProject', disks)
   const appsByProject = _.groupBy('googleProject', apps)
 
-  const getBillingProjectCell = (billingProject, shouldWarn) => {
+  const forAppText = appType => !!appType ? `for ${_.capitalize(appType)}` : ''
```

Comment:
Note that appType will be undefined for runtimes (Jupyter/RStudio). That's why I had the space before for originally, so that this would work for both apps and runtimes. A comment to this effect would be a good idea.